rust-escrow/dao-escrow/src/lib.rs

```diff
@@ -0,0 +1,185 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::collections::UnorderedMap;
+use near_sdk::{env, ext_contract, Gas, near_bindgen};
+use near_sdk::json_types::{Base64VecU8};
+use near_sdk::{AccountId, Balance, Promise, PromiseResult};
+
+/// Amount of gas for create action.
+pub const GAS_FOR_CREATE: Gas = Gas(90_000_000_000_000);
+pub const GAS_FOR_CREATE_CALLBACK: Gas = Gas(5_000_000_000_000);
+
+// define the methods we'll use on the other contract
+#[ext_contract(ext_dao_factory)]
+pub trait DaoFactory {
+    fn create(&mut self, name: String, args: Base64VecU8);
+}
+
+// define methods we'll use as callbacks on our contract
+#[ext_contract(ext_self)]
+pub trait MyContract {
```

Comment:
`MyContract`?

Shouldn't it be `DaoEscrow`?

Reply:
I had the same question but I think that my contract makes reference to this contract, the contract that your are building.